src/components/FacetResults.js

```diff
@@ -42,6 +43,10 @@ type FacetResultsProps = {
    * buckets. By default, facets with no buckets will be hidden.
    */
   showEmptyFacets: boolean;
+  /**
+   * An optional list of facets that should not be shown but are needed for other reasons
+   */
+   hideFacet: Array<string>;
```

Comment:
Add `?` to indicate this is optional. Make the prop name plural to indicate it's an array, not a boolean.
```suggestion
   hiddenFacets?: Array<string>;
```